01-filedir.md

```diff
@@ -155,135 +172,102 @@ which tells `ls` to add a trailing `/` to the names of directories:
+Desktop/      Downloads/    Movies/       Pictures/
 ~~~
 
 Here,
-we can see that `/Users/nelle` contains six **sub-directories**.
-The names that don't have trailing slashes,
-like `notes.txt`, `pizza.cfg`, and `solar.pdf`,
-are plain old files.
+we can see that our home directory contains mostly **sub-directories**.
+Any names in your output that don't have trailing slashes,
+are plain old **files**.
 And note that there is a space between `ls` and `-F`:
 without it,
 the shell thinks we're trying to run a command called `ls-F`,
 which doesn't exist.
 
-> ## What's In A Name? {.callout}
->
-> You may have noticed that all of Nelle's files' names are "something dot
-> something". This is just a convention: we can call a file `mythesis` or
-> almost anything else we want. However, most people use two-part names
-> most of the time to help them (and their programs) tell different kinds
-> of files apart. The second part of such a name is called the
-> **filename extension**, and indicates
-> what type of data the file holds: `.txt` signals a plain text file, `.pdf`
-> indicates a PDF document, `.cfg` is a configuration file full of parameters
-> for some program or other, and so on.
->
-> This is just a convention, albeit an important one. Files contain
-> bytes: it's up to us and our programs to interpret those bytes
-> according to the rules for PDF documents, images, and so on.
+> ## Parameters vs. Arguments {.callout}
 >
-> Naming a PNG image of a whale as `whale.mp3` doesn't somehow
-> magically turn it into a recording of whalesong, though it *might*
-> cause the operating system to try to open it with a music player
-> when someone double-clicks it.
+> According to [Wikipedia](https://en.wikipedia.org/wiki/Parameter_(computer_programming)#Parameters_and_arguments),
+> the terms argument and **parameter**
+> mean slightly different things.
+> In practice,
+> however,
+> most people use them interchangeably or inconsistently,
+> so we will too.
 
-Now let's take a look at what's in Nelle's `data` directory by running `ls -F data`,
+We can also use `ls` to see the contents of a different directory.  Let's take a 
+look at our `Desktop` directory by running `ls -F Desktop`,
 i.e.,
 the command `ls` with the **arguments** `-F` and `data`.
 The second argument --- the one *without* a leading dash --- tells `ls` that
 we want a listing of something other than our current working directory:
 
 ~~~ {.bash}
-$ ls -F data
+$ ls -F Desktop
 ~~~
 ~~~ {.output}
-amino-acids.txt   elements/     pdb/	        salmon.txt
-animals.txt       morse.txt     planets.txt     sunspot.txt
+shell-novice/
 ~~~
 
-The output shows us that there are four text files and two sub-sub-directories.
+Your output should be a list of all the files and sub-directories on your 
+Desktop, including the `shell-novice` directory you downloaded at 
+the start of the lesson.  Take a look at your Desktop to confirm that 
```

Comment:
Maybe add 'If you have many files the first ones listed may disappear out of view. You can use your mouse to scroll up to see them'